internal/common/common.go

```diff
@@ -144,6 +144,7 @@ func GetPowerScaleClusters(filePath string, logger *logrus.Logger) (map[string]*
 			cluster.Password,
 			cluster.IsiPath,
 			cluster.IsiVolumePathPermissions,
+			false,
```

Comment:
@baoy1 what's the new flag meaning here?

Reply:
`ignoreUnresolvableHosts`
I went through gopowerscale source code, looks like `ignoreUnresolvableHosts` won't take effect yet. The [default value](https://github.com/dell/gopowerscale/blob/main/api/api.go#L46) is false.